libco/libco.mk

```diff
@@ -5,17 +5,18 @@
 #
 # Include this make snippet to buid libco,a, a simple coroutine library.
 
-CHECK_LIBCO_FLAGS_MD5:=.libco_cflags-$(shell echo -- ${CFLAGS} ${CFLAGS_network} | shasum | sed 's/ *-//')
+LIBCO_DIR := $(abspath $(dir $(lastword ${MAKEFILE_LIST})))
+CHECK_LIBCO_FLAGS_MD5:=.libco_cflags-$(shell echo -- ${CFLAGS} | shasum | sed 's/ *-//')
 
 ${CHECK_LIBCO_FLAGS_MD5}:
```

Comment:
Should this `CHECK_LIBCO_FLAGS_MD5` also be in `LIBCO_DIR`

Reply:
Probably; I'll move it.

Reply:
Actually it should probably be ${CHECK_FLAGS_BOARD_MD5}  and not be built here.